### PR TITLE
DOCSP-33175 Remove 1.6 prerelease content

### PR DESCRIPTION
**Pull request by kennethdyer**
## Description

Removes 1.6 pre-release content.

## Staging

[Staging](https://preview-mongodbkennethdyer.gatsbyjs.io/cluster-sync/DOCSP-33175-remove-1.6-prerelease-content/)

## Jira

* [DOCSP-33175](https://jira.mongodb.org/browse/DOCSP-33175)

## Build

* [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6509efd6492df295cfdc0d9e)
* [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=650a0bc6492df295cfe56455)

### CODE REVIEW

**Review by mdb-ashley — APPROVED** [review]
This LGTM. Thanks, Kenneth!

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -32,16 +32,8 @@ General Limitations
   6.2. Only major MongoDB releases such as 6.0 or 7.0. are supported.
   For more information on MongoDB versioning, see
   :ref:`release-version-numbers`.
-- The source and destination clusters must have the same major and minor
```
Comment (ajayvijayakumar123) [inline source/reference/limitations.txt:35]: Could we revert the first bullet in the general limitations section to say `The minimum supported server version is MongoDB 6.0.5.` ?

File: source/reference/limitations.txt [hunk 2]
```diff
@@ -32,16 +32,8 @@ General Limitations
...
-  release version, but can have different patch releases.
-
-  For example,
-  - ``mongosync`` supports sync from a MongoDB 6.0.8 source cluster to a
-    destination cluster using MongoDB 6.0.9, since these are patch releases 
-    of the same major release.
-  - ``mongosync`` does not support sync from a MongoDB 6.0.9 source cluster to 
-    a destination cluster using MongoDB 7.0.0, since they have different major
-    versions.
+- The source and destination clusters must have the same major, minor, and
```
Comment (ajayvijayakumar123) [inline source/reference/limitations.txt:35]: Could we revert this to say `The source and destination clusters must have the same release
  version.` ?

File: source/reference/limitations.txt [hunk 3]
```diff
@@ -32,16 +32,8 @@ General Limitations
   6.2. Only major MongoDB releases such as 6.0 or 7.0. are supported.
   For more information on MongoDB versioning, see
   :ref:`release-version-numbers`.
-- The source and destination clusters must have the same major and minor
-  release version, but can have different patch releases.
-
```
Comment (ajayvijayakumar123) [inline source/reference/limitations.txt:37]: Can we revert the updates made to versioning.txt made from this [PR](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/160/files#diff-6c476215b8a109eb3828e67d96bdb157de4bce0f09d7470973baddf92baa1f76L63) ?

**Review by ajayvijayakumar123 — CHANGES_REQUESTED** [review]
Thanks! Reviewed the changes based on my input in https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/152 and https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/160. Will ask the team for further review on other components

File: source/reference/limitations.txt (hunk 3 above)
Comment (kennethdyer) [reply source/reference/limitations.txt:37]: Fixed

File: source/reference/limitations.txt (hunk 2 above)
Comment (kennethdyer) [reply source/reference/limitations.txt:35]: Fixed

File: source/reference/limitations.txt (hunk 1 above)
Comment (kennethdyer) [reply source/reference/limitations.txt:35]: Fixed

File: source/reference/collection-level-filtering.txt [hunk 4]
```diff
@@ -58,39 +44,11 @@ Filters have the following syntax:
...
-             "options": "<options>"
-          },
-          "collectionRegex": {
-             "pattern": "<regex-pattern>",
-             "options": "<options>"
-          }
        }
     ]
 
 Filters must include either the ``database`` field or the ``databaseRegex`` field.
```
Comment (jtcovan) [inline source/reference/collection-level-filtering.txt:50]: I think we'll want to revert this line too, since regex and exclusion are added together.

**Review by jtcovan — CHANGES_REQUESTED** [review]
Looks good to me % the regex database line

**Review by ajayvijayakumar123 — APPROVED** [review]
LGTM!

File: source/reference/collection-level-filtering.txt (hunk 4 above)
Comment (kennethdyer) [reply source/reference/collection-level-filtering.txt]: Fixed

**Pull request merged**